pennylane/ops/functions/iterative_qpe.py

```diff
@@ -15,21 +15,26 @@
 This module contains the qml.iterative_qpe function.
 """
 
+from warnings import warn
+
 import numpy as np
 
 import pennylane as qml
 
 
-def iterative_qpe(base, ancilla, iters):
+def iterative_qpe(base, aux_wire="unset", iters="unset", ancilla="unset"):
```

Comment:
It's mildly confusing that `iters` has `"unset"` as its unset specification, as it is expected to be an integer.
For me personally, `None` would be the more intuitive "unset value".

Reply:
I went with `"unset"` because originally `iters` was a positional argument. I feel like when someone opens the docs for `iterative_qpe`, if they spot `None` as the default, they might assume that they don't necessarily _need_ to set it and the function will choose a smart default on its own. Same reason as why I used `"unset"` for `aux_wire`'s default value. Happy to discuss further if you have any other questions.

Reply:
Fair enough :)